src/templates/ja/2019/base_chapter.html

```diff
@@ -2,15 +2,15 @@
 
 {% block title %}{{ metadata.get('title') }} | 2019 | HTTP ArchiveによるWeb年鑑{% endblock %}
 
-{% block description %}{{ metadata.get('description',metadata.get('title') + ' chapter of the ' + year + ' Web Almanac probing into the use of ' + metadata.get('description',metadata.get('title')) + ' on the web.') }}{% endblock %}
+{% block description %}{{ metadata.get('description',metadata.get('title') + ' の章 ' + year + ' Web年鑑の使用状況を調べる ' + metadata.get('description',metadata.get('title')) + ' をウェブ上で公開しています。') }}{% endblock %}
```

Comment:
I'm don't think we should translate Web Almanac. That's a brand name, as discussed here: https://github.com/HTTPArchive/almanac.httparchive.org/issues/567#issuecomment-559613733
```suggestion
{% block description %}{{ metadata.get('description',metadata.get('title') + ' の章 ' + year + ' Web Almanacの使用状況を調べる ' + metadata.get('description',metadata.get('title')) + ' をウェブ上で公開しています。') }}{% endblock %}
```

Reply:
OK, I understand.

But what about the chapters I've already translated (cms.md, http2.md, etc.)?
These chapters have been translated into the "Web年鑑".

Reply:
Yeah we should fix those too. Can you open a separate pull request for that?

Reply:
I can do a quick search and replace fairly easily if you want me to?

Reply:
Yes, I will create other branch and fix.